tests/EmitTest.elm

```diff
@@ -62,8 +81,7 @@ javascript =
                     [ ( "positive float", Literal (Float 12.3), "12.3" )
                     , ( "negative float", Literal (Float -12.3), "-12.3" )
                     , ( "positive zero float", Literal (Float 0.0), "0" )
-                    , -- Elm wat
-                      ( "negative zero float", Literal (Float -0.0), "0" )
+                    , ( "negative zero float", Literal (Float -0.0), "0" )
```

Comment:
Comment that was previously there is missing. I want the reader of the tests to notice that this is unexpected output, but -- that's how the official Elm compiler works currently.